src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1526,7 +1526,7 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
             {
                 InstanceId = instanceId,
                 WorkflowComponent = workflowComponent,
-                WorkflowType = workflowName //TODO: Change 'WorkflowType' to 'WorkflowName' once changes go through dapr/dapr
+                WorkflowName = workflowName
```

Comment:
This change and the one below it just come from updating the protos?

Reply:
Yes, updating the protos was failing the tests initially. On debugging I found i is due to these renaming in protos, so had to do here as well.